pylibdmtx/wrapper.py

```diff
@@ -397,3 +463,37 @@ def libdmtx_function(fname, restype, *args):
     DmtxPassFail,
     POINTER(POINTER(DmtxRegion))
 )
+
+dmtxImageGetProp = libdmtx_function(
+    'dmtxImageGetProp',
+    DmtxPassFail,
+    POINTER(DmtxImage),
+    c_int,  # prop
+)
+
+dmtxEncodeCreate = libdmtx_function(
+    'dmtxEncodeCreate',
+    POINTER(DmtxEncode),
+)
+
+dmtxEncodeDestroy = libdmtx_function(
+    'dmtxEncodeDestroy',
+    DmtxPassFail,
+    POINTER(POINTER(DmtxEncode))
+)
+
+dmtxEncodeSetProp = libdmtx_function(
+    'dmtxEncodeSetProp',
+    DmtxPassFail,
+    POINTER(DmtxEncode),
+    c_int,    # prop
+    c_int     # value
+)
+
+dmtxEncodeDataMatrix = libdmtx_function(
+    'dmtxEncodeDataMatrix',
+    DmtxPassFail,
+    POINTER(DmtxEncode),
+    c_int,
+    POINTER(c_char)
```

Comment:
My copy of `dmtx.h` (version 0.7.4) has the last argument as `unsigned char *`, which is represented by `c_ubyte` (https://docs.python.org/3/library/ctypes.html#fundamental-data-types). Could you check the type in your copy of the header?

Reply:
yes, that's right